modSecurity.bas

```diff
@@ -204,6 +204,48 @@ Function StripNulls(ByVal sStr As String) As String
     StripNulls = Left$(sStr, lstrlen(sStr))
 End Function
 
+Public Sub SavePassword(Password As String)

```

Comment:
This is not really secure.
Especially given that our community frequently sends their odyssey folders to each other, this presents an issue.
A short term solution would be to write the password out to a folder other than the odyssey one. Perhaps in the user's home directory or appdata.
Long term, we'll need some other solution.